### PR TITLE
Assess 'Accuracy' of model cards

### PR DESCRIPTION
**Pull request by fivegrant**
This PR creates a new entry called "Accuracy" that will render a new table on the dashboard. This only applies to model cards

### CODE REVIEW

File: tests/resources.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ profile_model:
   - text.json
   - paper.pdf
   - code.py
-  - amr.json
+  - ground_truth_model_card.json
```
Comment (brandomr) [inline tests/resources.yaml:19]: I think this should be optional

File: tests/test_e2e.py [hunk 2]
```diff
@@ -405,7 +404,7 @@ def test_profile_model(
     file_storage.upload("code.py", code)
 
     model_id = "test_profile_model"
-    amr = json.load(open(f"{context_dir}/amr.json"))
+    amr = json.load(open(f"./tests/amr.json"))
```
Comment (brandomr) [inline tests/test_e2e.py:407]: I think a real `AMR` could also optionally be provided here

File: tests/test_e2e.py [hunk 3]
```diff
@@ -457,6 +457,23 @@ def test_profile_model(
         status_response.json().get("status") == "finished"
     ), f"The RQ job failed.\n{job.latest_result().exc_string}"
 
+    #### POSTAMBLE ####
+    if not settings.MOCK_TA1:
```
Comment (brandomr) [inline tests/test_e2e.py:461]: add something like `and os.path.exists(f"{context_dir}/ground_truth_model_card.json")` ?

File: worker/operations.py [hunk 4]
```diff
@@ -517,14 +517,21 @@ def model_card(*args, **kwargs):
     text_file = paper_document_json.get(
         "text", "There is no documentation for this model"
     ).encode()
+    text_file = (
+        paper_document_json
+        .get("text", "There is no documentation for this model")
+    )
+
+    # TODO: Remove when no character limit exists for MIT
+    text_file = text_file[:9000]
```
Comment (brandomr) [inline worker/operations.py:526]: good TODO as a reminder.... i'll open an issue on MIT's repo so we can track this

File: tests/resources.yaml (hunk 1 above)
Comment (fivegrant) [reply tests/resources.yaml:19]: the only way we can make it optional atm is remove it from this yaml and do the optional check you suggested below (but that kind of defeats the purpose of resources.yaml right?)

File: tests/test_e2e.py (hunk 3 above)
Comment (fivegrant) [reply tests/test_e2e.py:461]: very doable. see above comment though

File: tests/test_e2e.py (hunk 2 above)
Comment (brandomr) [reply tests/test_e2e.py:407]: Ah yes, you are right! The AMR is irrelevant here it's just something we patch to. I'd forgotten it's not _really_ used by MIT

File: tests/resources.yaml (hunk 1 above)
Comment (brandomr) [reply tests/resources.yaml:19]: Maybe the `resources.yaml` file should have an `optional` section for various tests. Let's discuss a bit during standup

Comment (brandomr) [reply tests/resources.yaml]: done

**Review by brandomr — APPROVED** [review]

**Pull request merged**